self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -2065,25 +2501,31 @@
   },
   {
    "cell_type": "markdown",
-   "id": "b16d5016",
+   "id": "90370413",
```

Comment:
I suggest to include a lightweight cleanup section with `endpoint.delete()` , because otherwise an endpoint occurs ongoing charges (as its minimum min_replica_count is 1)